package/MDAnalysis/analysis/align.py

```diff
@@ -252,7 +252,7 @@ def rotation_matrix(a, b, weights=None):
     # so that R acts **to the left** and can be broadcasted; we're saving
     # one transpose. [orbeckst])
     rmsd = qcp.CalcRMSDRotationalMatrix(a.T, b.T, N, rot, weights)
-
+    logger.info('rmsd: {0}'.format(rmsd))
```

Comment:
nah to much logging. You are using that result right away. Keep the log for important information and don't spam it.